packages/analytics-client-common/src/session-manager.ts

```diff
@@ -4,7 +4,6 @@ import { getCookieName as getStorageKey } from './cookie-name';
 export class SessionManager implements ISessionManager {
   storageKey: string;
   cache: UserSession;
-  isSessionCacheValid = true;
```

Comment:
why remove this? cc @kevinpagtakhan do you know if this one is safe to remove?

Reply:
`isSessionCacheValid`  was used only in `getSessionId`/`setSessionId` logic and not used in others (`getDeviceId`/`setDeviceId`, `getUserId`/`setUserId`, `getLastEventTime`/`setLastEventTime`, etc)

Reply:
Maybe we can keep it if it doesn't hurt the debugging purpose, or you can move forward if you believe this doesn't break anything

Reply:
I believe it shouldn't break anytihng